RELEASE/scripts/autoscend/iotms/mr2019.ash

```diff
@@ -511,13 +506,13 @@ boolean auto_spoonReadyToTuneMoon()
 		}
 	}
 
-	if(isCanadia && !toCanadia && item_amount($item[logging hatchet]) == 0)
+	if(inCanadiaSign() && !toCanadia && item_amount($item[logging hatchet]) == 0)
 	{
 		// want to make sure we've grabbed the logging hatchet before switching away from canadia
 		return false;
 	}
 
-	if(isGnomad && !toGnomad && auto_is_valid($skill[Torso Awaregness]) && !auto_have_skill($skill[Torso Awaregness]))
+	if(inGnomeSign() && !toGnomad && auto_is_valid($skill[Torso Awaregness]) && !auto_have_skill($skill[Torso Awaregness]))
```

Comment:
```suggestion
	if(gnomads_available() && !toGnomad && auto_is_valid($skill[Torso Awaregness]) && !auto_have_skill($skill[Torso Awaregness]))
```

Reply:
that suggestion will give incorrect results in koe or if we did not unlock the desert yet